tika/tika.go

```diff
@@ -29,6 +29,16 @@ import (
 	"golang.org/x/net/context/ctxhttp"
 )
 
+// ClientError represents an error response from the Tika server.
+type ClientError struct {
+	// StatusCode is the http status code returned by the Tika server.
+	StatusCode int
+}
+
+func (e ClientError) Error() string {
+	return fmt.Sprintf("response code %d", e.StatusCode)
+}
+
 // Client represents a connection to a Tika Server.
```

Comment:
Would you mind expanding this comment to include a short example of using `ClientError` and `errors.As`? That way, people can find out this feature is available even if they aren't very familiar with `errors.As`.

Reply:
Good idea. What do you think about the change in 5a0674f?